### PR TITLE
LG-7819: Implement fake optInIPPApplicant endpoint

### PR DESCRIPTION
**Pull request by NavaTim**
## 🎫 Ticket

[LG-7819](https://cm-jira.usa.gov/browse/LG-7819)

## 🛠 Summary of changes

- Create fake USPS IPPaaS API endpoint
  - `/ivs-ippaas-api/IPPRest/resources/rest/optInIPPApplicant`
    - Successful response

This endpoint is used during the creation of IPP applicants. The implementation of this fake endpoint can help ensure that any supplementary UI-based load testing for IPP can (at least partially) proceed in the Team Joy environment while the USPS endpoint is being faked.

## 📜 Testing Plan

```sh
# Start fake server in background
make run &

# Try fake proofing results endpoint
curl -v -X POST http://127.0.0.1:5555/ivs-ippaas-api/IPPRest/resources/rest/optInIPPApplicant

# Output should show that result status is 200, type is application/json, and matches fixture

# Stop fake server
kill %%
```

### CODE REVIEW

File: spec/usps_spec.rb [hunk 1]
```diff
@@ -34,6 +34,34 @@
         end
     end
 
+    describe 'optInIPPApplicant' do
```
Comment (zachmargolis) [inline spec/usps_spec.rb:37]: this whole file is indented by 4? but we usually do by 2. Let's clean up the whole file in a separate PR?

**Review by zachmargolis — APPROVED** [review]
LGTM

File: spec/usps_spec.rb (hunk 1 above)
Comment (NavaTim) [reply spec/usps_spec.rb:37]: Yeah, I usually run the linting tool to clean things up but didn't notice a particular configuration for this repo.

Comment (NavaTim) [reply spec/usps_spec.rb:37]: @zachmargolis PR created: https://github.com/18F/identity-fake-server/pull/34

**Pull request merged**